### ISSUE TITLE
lint:accessibility throws error in clean standard installation

### ISSUE THREAD

**Issue by AdictoChannel** [namics/generator-nitro#127]
## Type of issue

* [x] Bug report
* [ ] Feature request
* [ ] Documentation issue or request

The issue refers to:

* [ ] The generator
* [x] A generated nitro project

## Environment

macOS High Sierra 10.13.3
Node.js v8.9.4
darwin 17.4.0
npm 5.6.0
generator-nitro 3.1.3

## Expected behavior

Linter runs throug without errors

## Current behavior

```
error An unexpected error occurred: "Command failed.\nExit code: 1\nCommand: sh\nArguments: -c gulp lint-accessibility\nDirectory: ...
ERROR: "lint:accessibility" exited with 1.
```

## Steps to reproduce the behavior

Clean install of nitro
`yo nitro`
Run linters
`yarn lint`

**Comment by ernscht** [comment]
You are right. Thank you for this report.

I will remove the exit process.exit part to make it work again.
This will change to not failing the lint task on errors.

Generally i'm not happy with the chosen lint:accessibility solution and the gulp plugin seems to have some issues.
We're looking for alternatives #120. Any help is very welcome ;-)

**Comment by ernscht** [comment]
published version 3.1.4 with a workaround ;-)
https://www.npmjs.com/package/generator-nitro
